### PR TITLE
Add a `!remind` command

### LINKED ISSUE #30: Repetitive Reminders (Deprecate ReminderBot)
Many newcomers express interest in daily reminders, but the Reminder bot doesn't allow for new people to set reminders for themselves in #accountability-station anymore. Horace should take up that role, and allow people to set repetitive reminders for themselves using some kind of command.

### PR DESCRIPTION
**Pull request by pralkarz**
Resolves #30 

This has been a long time coming, but it's there. It's pretty rough around the edges, especially regarding the aesthetics of the related messages, but that's the main purpose of this PR – to gather your feedback and discuss what would feel nice. 

As to the unit tests, I think they are better suited for another issue as there's a huge amount of mocking and stubbing needed to do, and I don't quite feel like working on that. Because of that the only real way to test the feature is to run `!remind help` and try out the commands one by one. I feel like I've caught a lot of the edge cases and random errors, but it's pretty much impossible not to miss anything.

Can't wait to hear from you!

P.S. I don't know why some of the commits from my previous pull request (#167) got there since they were merge. A helping hand would be appreciated!

### CODE REVIEW

File: commands/remind.js [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+	}
+
+	if (args.length === 0 || args.length === 1 && (args[0] === 'help' || args[0] === 'info')) {
+		const remindHelp = new Discord.RichEmbed()
+			.setColor('#FFEC09')
+			.setTitle(`${config.emotes.reminders} Knights of Academia Remind Help ${config.emotes.reminders}`)
+			.setDescription('Here are some commands to help you out with reminders!')
+			.addField('Add a reminder',
+				`\`!remind [me to] <task> in <how many> minutes/hours/days/months\`
+				Example: \`!remind me to do laundry in 2 hours\`
```
Comment (ghost) [inline commands/remind.js:29]: It might be easier to have these separated as fields.

**Review by ghost — APPROVED** [review]
Other than that one comment I'd say this is approved! Shoot me a ping here or on Discord when you want this to be merged.

File: commands/remind.js (hunk 1 above)
Comment (pralkarz) [reply commands/remind.js:29]: As discussed on Discord we can leave it like that for now and split it later down the line if more options are added. ^^
